src/AppInstallerCLICore/COMContext.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+#include "pch.h"
+#include "..\AppInstallerCommonCore\Public\AppInstallerProgress.h"
+#include "ExecutionContext.h"
+
+namespace AppInstaller
+{
+	using namespace AppInstaller::CLI;
+
+    enum ReportType
+    {
+        BeginProgress,
+        Progressing,
+        EndProgress,
+        ExecutionPhaseUpdate,
+    };
+
+	struct COMContext : IProgressSink, Execution::Context
+	{
+		COMContext(std::ostream& out, std::istream& in) : Execution::Context(out, in) 

```

Comment:
>std::ostream& out, std::istream& in [](start = 13, length = 35)

I feel like these are going to always be well known for the COM case.  Null streams for now, but in the future something else that we implement and hook up.  Maybe we will pass in the stream in this same way in the future, but I think for now we should just have an empty parameter list.

Reply:
In next iteration, I implemented both constructors - one which will pass nullstreams to context and another will just pass the inputs it gets to constructor. I can remove the latter if you say so.